core/lib/refinery/core/engine.rb

```diff
@@ -79,15 +79,39 @@ def refinery_inclusion!
         ].flatten
       end
 
-      # set the manifests and assets to be precompiled
-      # refinery/icons/* may still be used by plugins
+
+      # set assets to be precompiled
       config.to_prepare do
+
+        # scripts
         Rails.application.config.assets.precompile += %w(
-          refinery/*
+          refinery/*.js
           refinery/icons/*
           modernizr-min.js
           admin.js
         )
+
+        # stylesheets
+        Rails.application.config.assets.precompile += %w(
+          refinery/refinery.css
```

Comment:
@bricesanchez I didn't see this line note till today.
We don't allow *.css because we are using SASS style stylesheets and partials.
The stylesheet (in this case `refinery.scss`) uses `@import` to import the partials which make up the complete stylesheet. 
If you use `refinery/*.css` the precompiler will try to compile the partials on their own, leading to errors.

An extension should add its own assets to the precompile array, and you will automatically acquire them.